credoai/artifacts/data/comparison_data.py

```diff
@@ -55,7 +57,7 @@ def _validate_pairs(self):
             """Basic validation for pairs"""
```

Comment:
In line comments are in the format `# Your comment`, the triple quotes are reserved for docstrings.